src/validation.cpp

```diff
@@ -971,9 +969,9 @@ bool MemPoolAccept::AcceptSingleTransaction(const CTransactionRef& ptx, ATMPArgs
     GetMainSignals().TransactionAddedToMempool(ptx, nAcceptTime);
 
     const CTransaction& tx = *ptx;
-    boost::posix_time::ptime finish = boost::posix_time::microsec_clock::local_time();
-    boost::posix_time::time_duration diff = finish - start;
-    statsClient.timing("AcceptToMemoryPool_ms", diff.total_milliseconds(), 1.0f);
+    auto finish = std::chrono::system_clock::now();
+    auto diff = finish - start;
+    statsClient.timing("AcceptToMemoryPool_us", count_microseconds(diff), 1.0f);
```

Comment:
hmm... I think this is going to break stats on already running instances (if there any). Should keep string the same and use `count_milliseconds` instead imo. (same for similar changes below)

Reply:
If I change it to count_milliseconds it doesn't compile; I'd probably need to duration_cast it to milliseconds or something like that. Do you think would be good?

Reply:
We could do it like 4b57881d0d maybe?

>Class std::chrono::steady_clock represents a monotonic clock. The time points of this clock cannot decrease as physical time moves forward and the time between ticks of this clock is constant. This clock is not related to wall clock time (for example, it can be time since last reboot), and is most suitable for measuring intervals.

https://en.cppreference.com/w/cpp/chrono/steady_clock

Reply:
Yeah, that seems pretty reasonable to me